docker/postgres-setup/Dockerfile

```diff
@@ -10,10 +10,10 @@ WORKDIR /go/src/github.com/jwilder/dockerize
 RUN go install github.com/jwilder/dockerize@$DOCKERIZE_VERSION
 
 FROM alpine:3
+COPY --from=binary /go/bin/dockerize /usr/local/bin
```

Comment:
Doesn't change anything, just making this look more the like mysql-setup dockerfile